cmd/get_logs.go

```diff
@@ -362,6 +365,17 @@ func (glc *getLogsCmd) getCloudName() string {
 	return ""
 }
 
+func (glc *getLogsCmd) getControlPanelNodes() []v1.Node {
```

Comment:
`computeControlPanelNodes` may be slightly better

Reply:
Updated